app/src/views/features/sessions/SessionViewer/SaveRecordingConfigPopup.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { useLocation } from "react-router-dom";
+import { CloseOutlined } from "@ant-design/icons";
+import { Button, Checkbox, Radio, RadioChangeEvent, Row } from "antd";
+import { SessionSaveMode } from "./types";
+import { CheckboxValueType } from "antd/lib/checkbox/Group";
+
+interface Props {
+  isSaving?: boolean;
+  sessionSaveMode: SessionSaveMode;
+  includedDebugInfo: CheckboxValueType[];
+  setIncludedDebugInfo: (mode: CheckboxValueType[]) => void;
+  onClose: (e: React.MouseEvent) => void;
+  isIncludeNetworkLogsDisabled: boolean;
+  isIncludeConsoleLogsDisabled: boolean;
```

Comment:
We should move the `includedDebugInfo` state inside this component instead of passing as props. This is also cleanup `isIncludeNetworkLogsDisabled` as they are not required to be passed as props